doc/bootloader-building.rst

```diff
@@ -79,6 +79,19 @@ You can run the following to install everything required:
 
 * For other Distributions please consult the distributions documentation.
 
+If your distribution doesn't already have it you will also have to
+install ``zlib-devel``.
+
+* On Debian- or Ubuntu-like systems::
+
+    sudo apt-get install zlib1g-dev
```

Comment:
This "1g" looks like a version. Is there a more generic name available? On my RPM system the resp. package provides:
```
$ rpm -q --provides lib64zlib-devel
devel(libz(64bit))
lib64zlib-devel = 1.2…
lib64zlib-devel(x86-64) = 1.2…
libz-devel = 1.2…
libzlib-devel = 1.2…
pkgconfig(zlib) = 1.2…
zlib-devel = 1.2…
```
Does debian provide such aliases, too?

Reply:
> This "1g" looks like a version

Usually this is done to support multiple versions concurrently, that expose different APIs. I don't have access to a Debian system, but I would bet there were other suffixes too, as dependencies for different packages.

Reply:
Good point. I'll merge the install lines.

Reply:
And no, I don't think so. 

<details><summary>This is what I get if I search:</summary>


```shell
bwoodsend@ostrich:~$ apt search --names-only zlib
Sorting... Done
Full Text Search... Done
clisp-module-zlib/eoan 1:2.49.20180218+really2.49.92-3build3 amd64
  GNU CLISP module that adds zlib support for vectors

erlang-p1-zlib/eoan 1.0.6-1 amd64
  erlang interface to zlib

gambas3-gb-compress-bzlib2/eoan 3.13.0-1ubuntu2 amd64
  Gambas bzlib2 component

gambas3-gb-compress-zlib/eoan 3.13.0-1ubuntu2 amd64
  Gambas zlib compression component

gauche-zlib/eoan 0.9.6-10 amd64
  zlib binding for Gauche

libcompress-raw-zlib-perl/eoan 2.087-1 amd64
  low-level interface to zlib compression library

libghc-bzlib-dev/eoan 0.5.0.5-8build1 amd64
  Haskell bindings to the bzip2 library

libghc-bzlib-doc/eoan,eoan 0.5.0.5-8build1 all
  Haskell bindings to the bzip2 library; documentation

libghc-bzlib-prof/eoan 0.5.0.5-8build1 amd64
  Haskell bindings to the bzip2 library; profiling libraries

libghc-pipes-zlib-dev/eoan 0.4.4.2-2build1 amd64
  Zlib compression and decompression for Pipes streams

libghc-pipes-zlib-doc/eoan,eoan 0.4.4.2-2build1 all
  Zlib compression and decompression for Pipes streams; documentation

libghc-pipes-zlib-prof/eoan 0.4.4.2-2build1 amd64
  Zlib compression and decompression for Pipes streams; profiling libraries

libghc-zlib-bindings-dev/eoan 0.1.1.5-9build1 amd64
  low-level bindings to zlib

libghc-zlib-bindings-doc/eoan,eoan 0.1.1.5-9build1 all
  low-level bindings to zlib; documentation

libghc-zlib-bindings-prof/eoan 0.1.1.5-9build1 amd64
  low-level bindings to zlib; profiling libraries

libghc-zlib-dev/eoan 0.6.2-2build1 amd64
  Compression and decompression in the gzip and zlib formats

libghc-zlib-doc/eoan,eoan 0.6.2-2build1 all
  Compression and decompression in the gzip and zlib formats; documentation

libghc-zlib-prof/eoan 0.6.2-2build1 amd64
  Compression and decompression in the gzip and zlib formats; profiling libraries

libjzlib-java/eoan,eoan 1.1.3-2 all
  Java Zlib implementation

librte-pmd-zlib18.11/eoan-updates,eoan-security 18.11.5-0ubuntu0.19.10.2 amd64
  Data Plane Development Kit (librte-pmd-zlib runtime library)

librust-flate2+zlib-dev/eoan 1.0.6-1 amd64
  FLATE, Gzip, and Zlib bindings for Rust - feature "zlib"

lua-zlib/eoan 1.2-1 amd64
  zlib library for the Lua language

lua-zlib-dev/eoan 1.2-1 amd64
  zlib development files for the Lua language

node-browserify-zlib/eoan,eoan 0.2.0+20170820git8b3f0a862f6b+dfsg-3 all
  Full zlib module for browserify

r-bioc-zlibbioc/eoan,eoan 1.30.0+dfsg-1 all
  (Virtual) zlibbioc Bioconductor package

zlib-gst/eoan 3.2.5-1.3build1 amd64
  Zlib bindings for GNU Smalltalk

zlib1g/eoan,now 1:1.2.11.dfsg-1ubuntu3 amd64 [installed,automatic]
  compression library - runtime

zlib1g-dbg/eoan 1:1.2.11.dfsg-1ubuntu3 amd64
  compression library - development

zlib1g-dev/eoan,now 1:1.2.11.dfsg-1ubuntu3 amd64 [installed]
  compression library - development

zlibc/eoan 0.9k-4.3 amd64
  An on-fly auto-uncompressing C library
```

</details>

In my rather limited experience of Ubuntu, having these version numbers floating about is quite a normal thing.